types/block.go

```diff
@@ -587,9 +587,12 @@ const (
 const (
 	// Max size of commit without any commitSigs -> 82 for BlockID, 8 for Height, 4 for Round.
 	MaxCommitOverheadBytes int64 = 94
-	// Commit sig size is made up of 64 bytes for the signature, 20 bytes for the address,
+	// Commit sig size is made up of 96 bytes for the signature, 20 bytes for the address,
 	// 1 byte for the flag and 14 bytes for the timestamp.
-	MaxCommitSigBytes int64 = 109
+	MaxCommitSigBytes int64 = 141
```

Comment:
I am afraid the math here is wrong: 96 + 20 + 1 + 14 = 131.

Can we somehow test this?

Reply:
The math was wrong before, too. I don't know why exactly.